### PR TITLE
Add DevCycle Integration Tile

### PR DESCRIPTION
**Pull request by nikolasleblanc**
### What does this PR do?

This PR creates a DevCycle tile and associated metadata to be displayed on the DataDog integrations page.

### Motivation

This PR relates to [this PR](https://github.com/DataDog/documentation/pull/17953), in which docs where added to the DataDog RUM Feature Flag Integration docs to describe how to track DevCycle variable evaluations.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Not sure if I nailed the metadata properly so happy to take edits if required.

### CODE REVIEW

**Comment by buraizu** [comment]
Created DOCS-5501 to track.

**Comment by nikolasleblanc** [comment]
@buraizu @rtrieu what should I use as the codeowners entry? It looks like it needs write access to this repo but I don't imagine you're handing that out to just anybody.

**Comment by rtrieu** [comment]
@meghanlo Can you confirm that @nikolasleblanc added the information to the CODEOWNERS file in this repo correctly? Thank you!

File: devcycle/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# DevCycle integration
+
+## Overview
+
+DevCycle provides the following integrations with Datadog:
+
+### Feature flag tracking integration
+
+DevCycle's feature flag tracking integration enriches your RUM data with your feature's variable evaluations to provide visibility into performance monitoring and behavioral changes. Determine which users are shown a user experience and if it is negatively affecting the user's performance.
```
Comment (rtrieu) [inline devcycle/README.md:9]:
```suggestion
DevCycle's feature flag tracking integration enriches your RUM data with your feature's variable evaluations to provide visibility into performance monitoring and behavioral changes. Determine which users are shown a specific user experience and if it is negatively affecting the user's performance.
```

File: devcycle/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+## Setup
+
+### Feature flag tracking setup
+
+Feature flag tracking is available in the RUM Browser SDK. For detailed set up instructions, visit the [Getting started with Feature Flag data in RUM][4] guide.
```
Comment (rtrieu) [inline devcycle/README.md:15]:
```suggestion
Feature flag tracking is available in the RUM Browser SDK. For detailed setup instructions, visit the [Getting started with Feature Flag data in RUM][4] guide.
```

File: devcycle/images/IMAGES_README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Marketplace Media Carousel Guidelines
+
+## Using the media gallery
+
+To use the media gallery, you must upload a minimum of one image. The gallery
+can hold a maximum of 8 pieces of media total, and one of these pieces of media
+can be a video (guidelines and submission steps below). Images should be
```
Comment (rtrieu) [inline devcycle/images/IMAGES_README.md:7]:
```suggestion
can be a video (see guidelines and submission steps below). Images should be
```

File: devcycle/images/IMAGES_README.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+Description     : 300 characters maximum
+```
+
+### Video
+
+To display a video in your media gallery, please send our team the zipped file
+or a link to download the video at `marketplace@datadog.com`. In addition,
+please upload a thumbnail image for your video as a part of the pull request.
+Once approved, we will upload the file to Vimeo and provide you with the
+vimeo_id to add to your manifest.json file.  Please note that the gallery can
```
Comment (rtrieu) [inline devcycle/images/IMAGES_README.md:36]:
```suggestion
Vimeo ID to add to your `manifest.json` file.  Please note that the gallery can
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
A few minor suggestions, but otherwise looks good!

**Comment by nikolasleblanc** [comment]
Thanks @rtrieu, suggestions submitted!

**Review by rtrieu — APPROVED** [review]
lgtm

**Comment by nikolasleblanc** [comment]
Thanks @rtrieu, @emarsha94 anything I can do about that CODEOWNERS file?

**Review by yzhan289 — CHANGES_REQUESTED** [review]
Just 1 small fix!

File: .github/CODEOWNERS [hunk 5]
```diff
@@ -44,6 +44,7 @@
 /cyral/                                  @tyrannosaurus-becks product@cyral.com @DataDog/marketplace-review
 /data_runner/                            @DataDog/apps-sdk @DataDog/marketplace-review
 /datazoom/                               @DataDog/web-integrations
+/devcycle                                @nikolasleblanc
```
[the remark this answers is not in the document]
Comment (nikolasleblanc) [reply .github/CODEOWNERS:47]: Gah! There it is! Thanks!!

**Comment by nikolasleblanc** [comment]
Hey @yzhan289 anything more I can do for this PR?

**Review by yzhan289 — APPROVED** [review]
Hi @nikolasleblanc, this PR LGTM, but before the tile will show up on the list of integrations, we need to sync this internally which will take some time (~ less than 2 weeks).

**Comment by yzhan289** [comment]
Hm it doesn't look like I can merge this PR since it has some merge conflicts. Can you resolve them and then ping me again?

**Comment by nikolasleblanc** [comment]
@yzhan289 fixed!

**Comment by nikolasleblanc** [comment]
Hey @yzhan289 - friendly ping!

**Review by yzhan289 — APPROVED** [review]
LGTM!

**Comment by nikolasleblanc** [comment]
Hey @yzhan289 and @rtrieu one of my colleagues noticed typos in our code sample so I submitted a very small correction here: https://github.com/DataDog/integrations-extras/pull/1869

**Pull request merged**